.github/workflows/update_submodules.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Creates a PR to update TF & LLVM submodules when TF moves to a new LLVM commit.
+
+name: Update Submodules
+
+on:
+  schedule:
+     # Every weekday at 14:00 UTC (06:00 PST/07:00 PDT)
+     - cron: '0 14 * * 1-5'
```

Comment:
What happens if there is no diff? (e.g. no changes to TensorFlow or LLVM in 24h.... which seems unlikely)

Reply:
I think it just exits early successfully, but not totally sure.